resources/views/checkout/partials/sections/success/order-info.blade.php

```diff
@@ -1,3 +1,24 @@
 <x-rapidez-ct::card.inactive>
-    Hier moet de bezorg en factuur adres + gebruikte betaalmethode + gebruikte verzendmethode komen
+    <div class="flex">
```

Comment:
flex without wrap? shouldnt the shipping and billing cards be underneath eachother? not next to eachother, and if so, you can remove the flex div and add a class on the card.component